### PR TITLE
The next web clone

### PR DESCRIPTION
**Pull request by Ralph-1**
This Project is the clone of the Next Web website. It demonstrates the use of

- CSS grids
- Flexbox
- Floats
- Font Awesome Icons
- Media queries

It also has cleared linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,468 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
```
Comment (jubaan) [inline index.html:7]: - [ ] Notice how the correct paths start with a `./` dot
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">

// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">
```

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] Notice how the correct paths start with a `./` dot
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">

// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">
```

- [ ] The column menu only appears until it has reach tablet view 768px and bellow.
image

- [ ] Text in some elements still surpases its container
image

- [ ] `display: none` is being used in `.first-nav` please remove the second and use the class `.none` for it instead of duplicating the code. Keep code DRY.

### [Comments] :green_circle: 

- Great that you have changed the branch name but there is no need to open a new Pull Request, the branch can be changed in the previous PR. For next time use this link to know how to do it. [LINK](https://github.blog/2016-08-15-change-the-base-branch-of-a-pull-request/)

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+node_modules/
+.hintrc
+.stylelintrc.json
+package.json
+package-lock.json
```
Comment (descholar-ceo) [inline .gitignore:5]: Remove these files from `.gitignore`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Ralph-1 and @AshakaE  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [ ] These requested changes are not implemented yet

> # Status: Changes Required 
> There are some issues that need to be addressed before we can move forward. Please review the list below.
> 
> ## Required changes 
> ### [Mandatory] 
> * [ ]  Notice how the correct paths start with a `./` dot
> 
> ```
> // Bad Paths
> <link rel="stylesheet" href="/assets/stylesheets/main.css">
> <link rel="stylesheet" href="/assets/js/bootstrap.min.js">
> 
> 
> // Good Paths
> <link rel="stylesheet" href="./assets/stylesheets/main.css">
> <link rel="stylesheet" href="./assets/js/bootstrap.min.js">
> 
> ```
> 
> * [ ]  The column menu only appears until it has reach tablet view 768px and bellow.
>   image
> * [ ]  Text in some elements still surpases its container
>   image
> * [ ]  `display: none` is being used in `.first-nav` please remove the second and use the class `.none` for it instead of duplicating the code. Keep code DRY.
> 
> ### [Comments] 
> * Great that you have changed the branch name but there is no need to open a new Pull Request, the branch can be changed in the previous PR. For next time use this link to know how to do it. [LINK](https://github.blog/2016-08-15-change-the-base-branch-of-a-pull-request/)
> 
> ## Keep up the great work  and Happy Coding  !!


Kindly implement everything and submit for re-review. 

If anything seems to be unclear to you, comment on this Pull Request and mention me as [@descholar-ceo](https://github.com/descholar-ceo)

 | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by abredi — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @Ralph-1 and @AshakaE, Congratulations 🎊 👏, you did very well on this project.
Now, you can move ahead.

Please don't forget to merge to master. Good luck with the next projects!

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Pull request merged**